tutorials/cloudsdk-e2e-test-secure-applications/cloudsdk-e2e-test-secure-applications.md

```diff
@@ -14,15 +14,12 @@ primary_tag: products>sap-cloud-sdk
 ## Prerequisites
 This tutorial assumes you already completed the tutorial [End-to-End Tests for SAP Cloud SDK Projects](cloudsdk-e2e-test) and understood the concepts explained in the tutorial [Secure your application](s4sdk-secure-cloudfoundry).
 
-
-
-
-## Intro
 > ## We migrate tutorials to our [documentation](https://sap.github.io/cloud-sdk/)
```

Comment:
Do not header formatting to emphasize text. \*\*We migrate...\*\* would be correct in this case.